src/ToDo.jsx

```diff
@@ -1,32 +1,72 @@
 import React from 'react';
-import styles from './styles/ToDo.module.css';
+
+import { Stack, Button, styled } from '@mui/material';
+
+import theme from './Theme';
+
+const priorities = {
+    LOW: 'low',
+    MEDIUM: 'medium',
+    HIGH: 'high',
+};
 
 function ToDo({ toDo, onRemoveTodo }) {
     const handleButtonPress = (e) => {
         e.preventDefault();
         onRemoveTodo(e.currentTarget.parentNode);
     };
 
-    let className = styles.toDo;
-    if (toDo.new) {
-        className = styles.newToDo;
+    /*    className={clsx(styles.toDo, {
+                [styles.newToDo]: !!toDo.new,
+                [styles.low]: toDo.priority === priorities.LOW,
+                [styles.medium]: toDo.priority === priorities.MEDIUM,
+                [styles.high]: toDo.priority === priorities.HIGH,
+            })} */
+    function priorityColor(props) {
+        if (props === priorities.LOW) return theme.palette.priority.low;
+        if (props === priorities.MEDIUM) return theme.palette.priority.medium;
+
+        return theme.palette.priority.high;
     }
+    const NewBox = styled(Stack)((props) => ({
+        backgroundColor: 'RGB(247, 202, 201)',
+        borderStyle: 'solid',
+        borderRadius: '25px',
+        padding: '10px',
+        width: '400px',
+        height: '30px',
+        margin: 'auto',
+        fontWeight: 'bold',
+        color: priorityColor(props.priority),
+        borderColor: props.isNew && theme.palette.priority.new,
+    }));
+
     return (
-        <div
+        <NewBox
             id={toDo.id}
             key={toDo.id + toDo.task}
-            className={className}
             name="todo"
             value={toDo.id}
+            priority={toDo.priority}
+            isNew={toDo.new}
+            direction="row"
+            justifyContent="space-between"
```

Comment:
stick to 1 style customization method... if you already created a styled component then move all other styling props there